tempto-core/src/main/java/com/teradata/tempto/internal/query/QueryResultValueComparator.java

```diff
@@ -238,14 +249,14 @@ private int timestampEqual(Object actual, Object expected)
         return -1;
     }
 
-    private static boolean isIntegerValue(Object value)
+    private static boolean isLongOrNarrower(Object value)
```

Comment:
I think `isLongOrNarrower` = `value instance of Long || isIntegerOrNarrower(value)`